src/modulefinder/sentry_modulefinder_linux.c

```diff
@@ -348,11 +349,14 @@ sentry__procmaps_module_to_value(const sentry_module_t *module)
 static void
 try_append_module(sentry_value_t modules, const sentry_module_t *module)
 {
-    if (!module->file.ptr || !is_elf_module(module->start)) {
+    if (!module->file.ptr) {
```

Comment:
Should you call `is_elf_module` in `sentry__procmaps_module_to_value` now?

Reply:
yes, absolutely!